.changeset/gold-cars-drop.md

```diff
@@ -0,0 +1,6 @@
+---
+'@primer/react-brand': minor
+---
+
+**breaking** InlineLink components no longer take a `size` prop, but will now inherit their size in all cases.
+InlineLinks now work inside of `Header` components.
```

Comment:
This is great. On breaking changes we usually issue a migration example to make it super obvious how to fix their inevitable type errors.

Could you add this bit in please? 👇 

```diff
+ <InlineLink size="100" />
- <InlineLink /> // size will be inherited from the parent
```